src/Serilog.Sinks.ApplicationInsights/ExtensionMethods/LogEventExtensions.cs

```diff
@@ -114,7 +114,6 @@ public static class LogEventExtensions
             var exceptionTelemetry = new ExceptionTelemetry(logEvent.Exception)
             {
                 SeverityLevel = logEvent.Level.ToSeverityLevel(),
-                HandledAt = ExceptionHandledAt.UserCode,
```

Comment:
I will #if around it but i believe the data is no longer saved into app insights you must log it as a custom property, understand wanting to keep the changes to a minimum though!

https://docs.microsoft.com/en-us/dotnet/api/microsoft.applicationinsights.datacontracts.exceptionhandledat